src/download.py

```diff
@@ -89,16 +93,20 @@ def req_url(dl_file, max_retry=5, headers=None, proxies=None):
     for i in range(max_retry):
         try:
             r = requests.get(url, headers=headers, proxies=proxies)
+            if r.text.startswith("You can only make 350 requests every 15min"):
+                raise RequestLimitReached("Request limit reached")
```

Comment:
It may be good to raise an error, however a better solution would be: if this happens, just wait for >15 min.  What do you think? 
As long as we can eventually download the repo, there should be no problem


CC: This is similar to the solution you proposed  @leehoon7

Reply:
Looks great to me! Should you edit the code of waiting 15 mins, or should I? Either is good with me!

Reply:
I don't have much time recently, so I would be grateful if you do :)

Reply:
PS: we need to make sure we know which files were previously downloaded and which were not, I guess the solution is not simply `time.sleep(15min)`

Reply:
> PS: we need to make sure we know which files were previously downloaded and which were not, I guess the solution is not simply `time.sleep(15min)`

I think this line of code will make sure that we will not download previously downloaded files
https://github.com/fedebotu/clone-anonymous-github/blob/36ce2563bf5f99e84388d8fb88c073a9212c6690/src/download.py#L57


I will edit this PR and submit to the branch ASAP.

Reply:
> I don't have much time recently, so I would be grateful if you do :)

Updated and tested, please review my PR.